packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -112,18 +119,30 @@ const AmountValue = ({
           fontFamily={numberFontFamily}
           as={isReactNative ? undefined : 'span'}
         >
-          {integer}.
+          {integer}
         </BaseText>
         <BaseText
-          marginLeft="spacing.1"
           fontWeight={weight}
           fontSize={affixFontSize}
           fontFamily={numberFontFamily}
           color={amountValueColor}
           as={isReactNative ? undefined : 'span'}
+          opacity={isAffixSubtle ? 0.5 : 1}
         >
-          {decimal || '00'}
+          .{decimal || '00'}
         </BaseText>
+
+        {isStrikethrough && (
+          <BaseBox
+            // @ts-expect-error - intentionally setting the border color to the color prop for this hacky strikethrough
+            borderBottomColor={amountValueColor}
+            borderBottomWidth={type === 'body' ? 'thin' : 'thicker'}
+            borderBottomStyle="solid"
+            position="absolute"
+            width="100%"
+            top="50%"
```

Comment:
will top 50% be always correct? we use rems and if we increase or decrease fontsize of browser then I hope the place of line doesn't changes?

Reply:
Yes, 50% will always be in the middle of the amount value.